src/components/Form/validation.js

```diff
@@ -0,0 +1,63 @@
+
+import * as Yup from 'yup'
+
+const schema = Yup.object().shape({
+
+    personName: Yup
+        .string()
+        .trim()
+        .required('personName.errors.required'),
+    contact: Yup.object().shape({
+        tab: 
+            Yup.ref('$tab'),
+        mail: Yup
+            .string()
+            .trim()
```

Comment:
trim() does not work when we submit the form with formik, you may consider using trim manually on submit

Reply:
done